deploy/environments/applications.yaml

```diff
@@ -58,7 +58,7 @@ spack:
     - py-morphio
     - py-morphology-repair-workflow
     - py-morph-tool
-    - py-neurodamus
+    - py-neurodamus ^neuron%intel
```

Comment:
I think this is needed as in https://github.com/BlueBrain/spack/pull/1066, but please confirm.